docs/architecture/ARCHITECTURE.md

```
@@ -162,8 +168,8 @@ project-root/
 ### 6.2. External System Integration
 
 **Synchronization Strategy**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)